pkg/dir.props

```diff
@@ -49,4 +49,9 @@
     <AdditionalSymbolPackageExcludes Condition="'$(LibraryFileExtension)' != ''" Include="%2A%2A\%2A.a;%2A%2A\%2A$(LibraryFileExtension)" />
   </ItemGroup>
   
+  <PropertyGroup>
+    <!-- BlockStable on private packages by default -->
+    <BlockStable Condition="'$(BlockStable)' == '' and $(MSBuildProjectName.Contains('Private'))">true</BlockStable>
```

Comment:
What's going to happen with CoreFx.Private.TestUtilities?  Maybe we should put this in packaging.props (or packaging.targets) instead?

Reply:
we can consider that but I think we can keep having those projects setting the property. Doing it general will also have some needing to opt-out like System.Private.ServiceModel.

Reply:
I see, that's fair.